<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addoredit_PovyshKval</class>
 <widget class="QDialog" name="Addoredit_PovyshKval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>добавление записи</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="17" column="0">
       <widget class="QPushButton" name="pushButton_ok">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ОК</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/ok.png</normaloff>:/img/img/ok.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите дату</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_error">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string>Заполните обязательные поля</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите сотрудника *</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите тему *</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_kurs">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_tema">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_sotr">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите место</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите  курс *</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_sert">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_mesto">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QPushButton" name="pushButton_choose">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выбрать</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/1134892.png</normaloff>:/img/img/1134892.png</iconset>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите сертификат</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите файл</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/cancel.png</normaloff>:/img/img/cancel.png</iconset>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLineEdit" name="lineEdit_file">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QDateEdit" name="dateEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="displayFormat">
         <string>dd MMMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите оценку</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_ocenka">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
